lib/subscription_client/resources.rb

```diff
@@ -34,7 +34,13 @@ def setup_resources
   end
 
   def find_resources
-    find_plugins
+    resources = find_plugins
```

Comment:
@merefield Could you explain the thinking behind this change? Looks like it's not changing the functionality and will break the use of TEST_SUBSCRIPTION_URL.

Reply:
Yes, without refactoring here and adding a layer, you can't stub the method acts as the directory search.

Reply:
Ah I see, that makes sense. Don't we want to keep TEST_SUBSCRIPTION_URL functionality too though?

Reply:
TEST_SUBSCRIPTION_URL is still respected.  I've just checked that on a standard install connecting to `test2`